usagov/vote-gov-drupal
# VOTE-2519 Show language sub-menu while main nav is open

## Jira ticket

[Vote-2519](https://cm-jira.usa.gov/browse/VOTE-2519)

## Description

Addressing bug to address issue with main menu being open and language menu not being able to be open as the same time. 

**Solution**
Targeting the mobile menu with the attribute with `aria-hidden="true"` and when it is not hidden

## Deployment and testing

### Post-deploy steps

1. cd into the `votegov` theme and run `npm run build`

### QA/Testing instructions

1. adjust the screen to mobile view `390` and toggle the main menu on while the menu is open also open the language section as well.  Expected behavior is that both can be open at the same time

## Checklist for the Developer




## Checklist for the Peer Reviewers
